triplea_maps.yaml

```diff
@@ -1815,6 +1815,15 @@
     <br>
     Ultimate World Variants
     <br>Includes several Ultimate World Variants including FFAs
+- mapName: House of Habsburg
+  mapCategory: EXPERIMENTAL
+  url: https://github.com/triplea-maps/house_of_habsburg/archive/master.zip
+  version: 1
+  img:
```

Comment:
The travis build failed because no URL has been specified here.
Either you add a valid URL here, or drop this line entirely.

Reply:
@CrazyGerman you can also post an image to a github issue or any github dialog, it'll give you a URL; you can use that URL here too if it's easy than checking in to the map repository.